Classes/Views/LabelCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  LabelCell.swift
+//  Freetime
+//
+//  Created by Sherlock, James on 15/07/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+final class LabelCell: UICollectionViewCell {
+    
+    let label = UILabel()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        contentView.backgroundColor = .clear
+        accessibilityTraits |= UIAccessibilityTraitStaticText
```

Comment:
Admittedly wouldn't have thought to do it if it wasn't for the fact I copied it from the button cell 😅 😅